src/components/header/style.module.css

```diff
@@ -481,16 +481,14 @@
 	}
 }
 
-:global {
```

Comment:
Whatever Vite's using to handle CSS Modules considers this a syntax error as it drops the `:global`, resulting in a bare `{`. Easy enough to fix.